yafti/app.py

```diff
@@ -33,6 +37,15 @@ def __init__(self, cfg: Config = None, loop=None):
     def run(self, *args, **kwargs):
         configured_mode = self.config.properties.mode
         _p = self.config.properties.path.expanduser()
+        ## ---- TODO: START: REMOVE THIS HACK AFTER ENOUGH PEOPLE HAVE UPGRADED ----
+        _old_p = Path("~/.config/yafti-last-run").expanduser()
+        if _old_p.exists():
+            if not _p.parent.is_dir():
+                _p.parent.mkdir(mode=0o755, parents=True, exist_ok=True)
+            if _p.is_file():
```

Comment:
I don't think we want to remove the current properties path if it exists. Instead I'd opt to log a warning that both old path and new path were found but to defer to the new file being created.